upgrade/steps_test.go

```diff
@@ -203,3 +203,37 @@ func TestEnsureUpstreamRepo(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+func TestReleaseLabel(t *testing.T) {
+	tests := []struct {
+		from, to string
+		expect   string
+	}{
+		{"v1.2.3", "v1.2.3", ""},
+		{"v1.2.3", "v1.2.4", "needs-release/patch"},
+		{"v1.1.3", "v1.2.4", "needs-release/minor"},
+		{"v2.2.3", "v1.2.4", "needs-release/major"},
```

Comment:
this is kind of an interesting case, because you're actually indicating a downgrade. I wonder if we should actually return nothing in a downgrade just to be safe.

Reply:
That's a good corner case to test. I like that behavior change.